components/molecules/CallToAction.js

```diff
@@ -46,9 +50,14 @@ CallToAction.propTypes = {
   title: PropTypes.string.isRequired,
 
   /**
-   * a short description about what the call to action is about
+   * a short description about what the call to action is about - part 1
+   */
+  descriptionP1: PropTypes.string.isRequired,
```

Comment:
To keep this component simple, I would recommend implementing the description this way:

1. In CallToAction.js, remove the `<div>` on line 16 that is wrapping the `<p>` (This div is not useful)
2. Add this to the `<p>` tag:
```html
<p style={{ whiteSpace: "pre-line" }}>{props.descriptionP1}</p>
```
3. Combine the two strings in the translation files and separate them with two newline characters `\n` in that single string like so: 

"You’re invited to test our experimental ideas and participate in research interviews to make Service Canada better for everyone. Every bit of feedback helps us make sure we’re making our services simple and easy to use.\n\nYour participation will not affect your access to government services. You can unsubscribe at any time."

Reply:
This way we can have 1 description prop and put whatever string length or paragraphs you want.

Reply:
Ya that makes much more sense. I was thinking about that but I did not know if we could do \n in the lang files